### ISSUE TITLE
FE source code improve

### ISSUE THREAD

**Issue by xuanlongvts** [metaplex-foundation/metaplex#345]
hi team.

https://github.com/metaplex-foundation/metaplex/blob/master/js/packages/web/src/routes.tsx

In this part, when we use NextJS, then we don't need the  react-router-dom

This is doc: https://nextjs.org/docs/basic-features/pages

**Comment by ProPratik1405** [comment]
Yes, you can replace this code by going to pages and then creating the path name files and then from inside the each pages file call the components they were rendering. This should solve the issue

**Comment by jordansexton** [comment]
Closing as this issue isn't actionable, but please open a PR if there's an improvement to the code you've made!
